### DISCUSSION TITLE
[Feature request] Scroll tabs with mouse wheel

### DISCUSSION THREAD

**Discussion by KParthSingh** [zen-browser/desktop#1897] [Ideas]
In Sublime Text, there’s a feature that’s very useful. When you have multiple tabs open, you can switch from one tab to another by simply placing the cursor over the tab area and scrolling the mouse wheel up and down. This allows you to quickly glance at the content of each tab without having to click on each tab one by one. It's a deal breaker for me I use it every time on edge with Windhawk script.

**Comment by Axenide** [comment]
This is an useful feature, I used to have this enabled with Sidebery.

**Reply by KParthSingh** [reply]
Thanks. I think it would be nice if they inbuilt it.

**Reply by KParthSingh** [reply]
As @cswimr has pointed out it's already in zen browser. so you don't have to use any third-party tool.

**Comment by cswimr** [comment]
FYI, this is already possible through a browser preference, see [Zen Browser FAQ - How to switch tabs by scrolling](https://docs.zen-browser.app/faq#how-to-switch-tabs-by-scrolling).
> You can enable this feature by changing a setting in the browser’s configuration. Here’s how:
> 
> 1. Open the `about:config` page. This page contains advanced settings for the browser.
> 2. Search for `toolkit.tabbox.switchByScrolling`
> 3. Toggle the setting to `true` by double-clicking on it
> 
> Once this setting is enabled, you can hover your mouse over the tab bar and use your mouse wheel to scroll through the tabs, making it easier to navigate between them.

**Reply by KParthSingh** [reply]
OH. Didn't knew it already exist. thanks pal.

**Reply by Axenide** [reply]
Wow, thanks!

**Comment by KParthSingh** [comment]
I was wondering why this is behind a flag? why don't you just add a toggle in settings?

**Comment by trapexit** [comment]
How about tab scrolling without it having to be within the tab area? By using a set of modifier keys and then scrolling. Would be far faster and less error prone for those who keep the tab area closed and don't have it up against the edge of their desktop which stops the mouse movement.

**Comment by ZeChArtiahSaher** [comment]
It would be nice if it worked with shortcuts like scroll + (shift + alt / shift + ctrl) or something to swap positions of tabs like in firefox
